src/main/AndroidManifest.xml

```diff
@@ -333,7 +333,8 @@ limitations under the License.
 
         <activity
             android:name=".TrackListActivity"
-            android:exported="true">
+            android:exported="true"
```

Comment:
This is necessary to make search work. Otherwise `onNewIntent()` will never be called. This option is there, so that Android doesn't create a new instance of TrackListActivity when a search action is performed, but reuses the currently existing one.